unicore/ask/service/questions_api.py

```diff
@@ -20,9 +21,35 @@ class QuestionResource(object):
     def __init__(self, request):
         self.request = request
 
-    @view(renderer='json')
+    @view(renderer='json', schema=QuestionSchema)
```

Comment:
is `schema=QuestionSchema` a cornice thing? Does it return nice responses if the validation fails?